core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAOSpec.scala

```diff
@@ -128,16 +128,25 @@ class BillingProfileManagerDAOSpec extends AnyFlatSpec with MockitoTestUtils {
 
   behavior of "createBillingProfile"
 
-  it should "fail when provided with Google billing account information" in {
+  it should "create a GCP profile in billing profile manager if given RawlsBillingAccountName" in {
```

Comment:
Nice that this was already a test case, makes it clear what test needs to change!